docs/source/export.rst

```diff
@@ -1,11 +1,293 @@
+.. _torch.export:
+
 torch.export
 =====================
 
-.. TODO: Add torch.export() tutorial here.
-
 .. warning::
     This feature is a prototype and may have compatibility breaking changes in the future.
 
+
+Overview
+--------
+
+``torch.export`` is an ahead-of-time (AOT) process built to fully convert
+define-by-run PyTorch code into a static and standardized model representation,
+with strong soundness guarantees. It is designed to streamline and standardize the
+way PyTorch interacts with various systems and platforms.
+
+``torch.export`` creates a full graph representation of a PyTorch model
+that can be saved and loaded and run in different environments and
+languages. In comparison, ``torch.compile`` is a JIT compiler which provides
+the flexibility of creating partial graphs, where untraceable parts of models
```

Comment:
Might be worth elaborating more on why an end user might want to use `torch.compile(m, fullgraph=True)` vs `torch.export()` 

Something along the lines of if you need your model to run in environments where python is not available you're going to have to remove graph breaks